core/app/finders/spree/orders/find_current.rb

```diff
@@ -4,12 +4,13 @@ class FindCurrent
       def execute(user:, store:, **params)
         params = params.merge(store_id: store.id)
 
-        order = incomplete_orders.find_by(params)
```

Comment:
@pmilewski please modify the `incomplete_orders` method - add a SQL scope to remoce cancelled there